core/src/main/scala/backbone/consumer/Consumer.scala

```diff
@@ -25,7 +26,8 @@ object Consumer {
       queueUrl: String,
       events: List[String],
       parallelism: Int = 1,
-      limitation: Option[Limitation] = None
+      limitation: Option[Limitation] = None,
+      sqsSourceSettings: SqsSourceSettings
```

Comment:
Please don't add the SqsSourceSettings directly there. We shouldn't have other libraries classes in our api.